test/simple_select.test.js

```diff
@@ -0,0 +1,348 @@
+/* eslint no-shadow:[0] */
+import test from 'tape';
+import GLDraw from '../';
+import { createMap, cloneFeature } from './test_utils';
+import spy from 'sinon/lib/sinon/spy'; // avoid babel-register-related error by importing only spy
+import stub from 'sinon/lib/sinon/stub'; // avoid babel-register-related error by importing only stub
+import AfterNextRender from './utils/after_next_render';
+import makeMouseEvent from './utils/make_mouse_event';
+
+test('simple_select', t => {
+
+  var map = createMap();
+  spy(map, 'fire');
+
+  var Draw = GLDraw();
+  map.addControl(Draw);
+
+  var afterNextRender = AfterNextRender(map);
+
+  var cleanUp = function(cb) {
+    Draw.deleteAll();
+    map.fire.reset();
+    afterNextRender(cb);
+  }
+
+  var getFireArgs = function() {
+    var args = [];
+    for (var i=0; i<map.fire.callCount; i++) {
+      args.push(map.fire.getCall(i).args);
+    }
+    return args;
+  }
+
+  t.test('simple_select - setup', t => {
```

Comment:
Ah, bad name. The goal of this is just to confirm that that map is ready. Pushed an update to make that more clear and do this in a better way.